client/app/pages/queries/components/QueryMetadata.less

```diff
@@ -0,0 +1,79 @@
+// ANGULAR_REMOVE_ME Change this class to `.query-metadata` when all related Angular code removed
```

Comment:
Noticing it now, it's better if we keep `.query-source-metadata` here and `.query-view-metadata` for the view page to avoid conflicts.

Reply:
@gabrieldutra I changed implementation of QueryMetadata component so now it can display data source too (https://github.com/getredash/redash/pull/4476/commits/32bd46e88d9a24770563bf9e2d82d6c8783ce6de). I temporarily added data source to Query Source page so you can try in on preview instance of this PR (I'll remove it before merging :slightly_smiling_face:). It slightly differs from all existing variants, but not too much and IMO works good enough (check how it stacks on different screen widths)

Reply:
Cool, it looks it can be applied in the View page with no issues :+1: